### DISCUSSION TITLE
Introductions 👋

### DISCUSSION THREAD

**Discussion by phipsae** [BuidlGuidl/batch10.buidlguidl.com#1] [Announcements]
### 👋 Welcome Batch 10 Builders!

Philip here, I've been a proud member of BuidlGuidl for the past year now. I am developer, love to learn everything about smart contracts and support the batches program. 

Hope that the batch will be a very fun project for you all to go through. If you have any questions, feel free to ask anytime!

**Comment by derrekcoleman** [comment]
Hi everyone! My name is Derrek, "batch dad" of the group. I was onboarded to web3 by [RaidGuild](https://www.raidguild.org/) three years ago and haven't looked back since. Looking forward to buidling with you all!

**Comment by Favvie** [comment]
Hi everyone! I'm Favour, a frontend developer eager to expand my expertise into smart contract development. I'm excited to collaborate and build innovative, impactful projects with this community. Looking forward to learning, growing, and creating together!

**Reply by derrekcoleman** [reply]
Welcome, Favour! I look forward to buidling with you too 🤝

**Reply by phipsae** [reply]
Welcome!

**Comment by Michael-Nwachukwu** [comment]
Hello everyone. My name is michael and I'm a frontend and smart contract developer, and a technical writer as well. I am looking forward to Buidling cool stuff with you all. Cheers.

**Reply by derrekcoleman** [reply]
Technical writers are rare - we're lucky to have you!

**Reply by Michael-Nwachukwu** [reply]
Glad to be here

**Comment by Superior212** [comment]
Hello everyone! My name is Samson, and I’m a full-stack developer and smart contract developer. I’m excited to collaborate and build amazing projects with all of you. Cheers🤗!

**Reply by phipsae** [reply]
Welcome, nice to have you!

**Reply by derrekcoleman** [reply]
Welcome to the batch, Samson! Looking forward to buidling with you 👨🏼‍💻

**Comment by aspiringLilCoder** [comment]
Hi Derrek, Philip and batchmates! I'm Cherry and currently pursuing a bachelor's degree in Computer Science. I'm in the process of trying to master smart contract development, so greatly look forward to be a part of this batch program and code along with you all :))

**Reply by derrekcoleman** [reply]
Welcome, Cherry! Mastering anything is a long journey; you're on the right path!

**Reply by phipsae** [reply]
Nice to have you!

**Comment by pbkompasz** [comment]
Hey everyone! I'm Peter, a full stack developer, and I'm excited for the opportunity to build with you all.

**Reply by derrekcoleman** [reply]
It's great to have you with us, Peter!

**Reply by phipsae** [reply]
Welcome!

**Comment by Lukman-01** [comment]
Hi everyone! My name is Lukman, I am a Smart contract developer Using Solidity and Cairo. I have experience in smart contract security and gas optimization, I am eager to expand my expertise into smart contract development. I'm excited to collaborate and build innovative. Looking forward to learning, growing, and creating together!

**Reply by derrekcoleman** [reply]
Welcome to the batch, Lukman! Lucky us to have someone with Cairo experience around 👀

**Reply by Lukman-01** [reply]
Nice to have you as our mentor...

**Reply by phipsae** [reply]
Welcome!

**Comment by seriffsans** [comment]
Hi everyone! I'm Kristine, and currently exploring the world of blockchain and web3. I'm a beginner in Solidity, and have some experience in web frontend development. Nice to build with everyone here!

**Reply by phipsae** [reply]
Glad you are here!

**Reply by derrekcoleman** [reply]
Hi Kristine! We're all lifelong beginners in one way or another, so you're among frens

**Comment by michojekunle** [comment]
Hello everyone. My name is Michael and I'm a Frontend, smart contract developer, and technical writer. I'm excited to be here and I am looking forward to collaborating and building with you all. 🥳🤗

**Reply by phipsae** [reply]
Welcome @michojekunle, nice to have you!

**Reply by derrekcoleman** [reply]
Welcome, Michael! So happy to have you joining us

**Reply by michojekunle** [reply]
> Welcome @michojekunle, nice to have you! 

Thank you 😊

**Reply by michojekunle** [reply]
> Welcome, Michael! So happy to have you joining us

Thank you 🤗

**Comment by emarc99** [comment]
Hy everyone!! Emmanuel, here. I do backend (Django), smart contract(solidity) and cyber security. Glad to join and build with you all.

**Reply by derrekcoleman** [reply]
Welcome, @emarc99! It's great to have you joining us 🙌

**Reply by phipsae** [reply]
Welcome! :)

**Comment by kcmikee** [comment]
Hey everyone! I'm KC, a frontend and smart contract developer. Excited to join this community and looking forward to building and collaborating with all of you!

**Reply by phipsae** [reply]
Glad to have you!

**Reply by derrekcoleman** [reply]
Welcome to the Batch, KC! 🙌

**Comment by Nick-Vitreus** [comment]
Hi Philip!

Great discussion here! 💡 One challenge we've noticed for many Web3 developers is the financial and operational barriers some networks create. Polkadot's parachain model, while innovative, can be a steep hurdle for smaller teams or projects that need flexibility without breaking the bank.

At Vitreus (check out our [Discord](https://discord.gg/VITREUS)), we’re rethinking blockchain infrastructure with a Layer-0 architecture designed to be accessible for everyone—no costly parachain auctions, no restrictive resource allocation. Developers can enjoy low to no gas fees and scale seamlessly, whether you're a solo dev launching a small dApp or a large team building the next big thing. It's all about empowering builders at every stage.

Would love to hear your thoughts on how inclusive infrastructure impacts innovation!
